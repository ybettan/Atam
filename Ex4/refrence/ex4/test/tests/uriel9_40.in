55400 40AOOOOOOOBOOROOOOOFORBOOOO0 0 D3 0 R3 3 U2 1 R2 1 L2 1 L4 4 U4 4 D3 4 U3 4 R3 4 L3 4 L3 4 L3 4 L0 0 D3 0 R3 3 U3 4 U3 4 R3 4 L3 4 D3 4 D3 4 L0 0 D3 0 R3 3 U3 4 L3 4 L0 0 D3 0 R3 3 U3 4 L