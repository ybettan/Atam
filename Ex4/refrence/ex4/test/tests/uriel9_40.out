Please enter number of rows:
5
Please enter number of columns:
5
Please enter game duration in seconds and max number of moves:
400 40
Please enter the board:
AOOOOOOOBOOROOOOOFORBOOOO

We are all set. Total time: 400 sec
Max number of allowed moves: 40 moves
Press Enter key when you are ready...

Current Board:
A * * * *
* * * B *
* R * * *
* * F * R
B * * * *
Moves:
40 moves left, please enter your move:
0 0 D

Current Board:
* * * * *
* * * B *
* R * * *
A * F * R
B * * * *
Moves: (0,0,D)
37 moves left, please enter your move:
3 0 R

Current Board:
* * * * *
* * * B *
* R * * *
* * F A R
B * * * *
Moves: (0,0,D)->(3,0,R)
36 moves left, please enter your move:
3 3 U

Current Board:
* * * * *
* * * B *
* R * A *
* * F * R
B * * * *
Moves: (0,0,D)->(3,0,R)->(3,3,U)
35 moves left, please enter your move:
2 1 R

Current Board:
* * * * *
* * * B *
* * R A *
* * F * R
B * * * *
Moves: (0,0,D)->(3,0,R)->(3,3,U)->(2,1,R)
34 moves left, please enter your move:
2 1 L

Illegal move. Try again
2 1 L

Illegal move. Try again
4 4 U

Illegal move. Try again
4 4 D

Illegal move. Try again
3 4 U

Illegal move. Try again
3 4 R

Illegal move. Try again
3 4 L

Illegal move. Try again
3 4 L

Illegal move. Try again
3 4 L

Illegal move. Try again
3 4 L

Illegal move. Try again
0 0 D

Illegal move. Try again
3 0 R

Illegal move. Try again
3 3 U

Illegal move. Try again
3 4 U

Illegal move. Try again
3 4 R

Illegal move. Try again
3 4 L

Illegal move. Try again
3 4 D

Illegal move. Try again
3 4 D

Illegal move. Try again
3 4 L

Illegal move. Try again
0 0 D

Illegal move. Try again
3 0 R

Illegal move. Try again
3 3 U

Illegal move. Try again
3 4 L

Illegal move. Try again
3 4 L

Illegal move. Try again
0 0 D

Illegal move. Try again
3 0 R

Illegal move. Try again
3 3 U

Illegal move. Try again
3 4 L

Illegal move. Try again

No more moves allowed. Game Over!
